gridt/src/app/app.component.html

```diff
@@ -1,28 +1,28 @@
 <ion-app>
-    <ion-menu contentId="main" side="start" menuId="first" *ngIf="isLoggedIn$ | async">
-      <ion-header>
-        <ion-toolbar>
-          <ion-title>Menu</ion-title>
-        </ion-toolbar>
-      </ion-header>
-      <ion-content>
-        <ion-list>
-          <ion-menu-toggle auto-hide="true" *ngFor="let p of appPages">
-            <ion-item [routerDirection]="'root'" [routerLink]="[p.url]">
-              <ion-icon slot="start" [name]="p.icon"></ion-icon>
-              <ion-label>
-                {{p.title}}
-              </ion-label>
-            </ion-item>
-          </ion-menu-toggle>
-        </ion-list>      
-      </ion-content>
-      <ion-footer>
-        <ion-button color="dark" expand="full" (click)="auth.logout()" href="/">
-          <ion-icon class="ion-padding-end" name="exit-outline"></ion-icon>
-          Logout
-        </ion-button>
-      </ion-footer>
-    </ion-menu>
-    <ion-router-outlet id="main"></ion-router-outlet>
+  <ion-menu contentId="main" side="start" menuId="first" *ngIf="isLoggedIn$ | async">
+    <ion-header>
+      <ion-toolbar>
+        <ion-title>Menu</ion-title>
+      </ion-toolbar>
+    </ion-header>
+    <ion-content>
+      <ion-list>
+        <ion-menu-toggle auto-hide="true" *ngFor="let p of appPages">
+          <ion-item [routerDirection]="'root'" [routerLink]="[p.url]">
+            <ion-icon slot="start" [name]="p.icon"></ion-icon>
+            <ion-label>
+              {{p.title}}
+            </ion-label>
+          </ion-item>
+        </ion-menu-toggle>
+      </ion-list>
+    </ion-content>
+    <ion-footer>
+      <ion-button color="dark" expand="full" (click)="auth.logout()" href="/">
+        <ion-icon class="ion-padding-end" name="exit-outline"></ion-icon>
+        Logout
+      </ion-button>
+    </ion-footer>
+  </ion-menu>
+  <ion-router-outlet id="main"></ion-router-outlet>
```

Comment:
Why is this file in the commit? There seem to be no changes.

Reply:
I believe the indentation was incorrect which I resolved here.